.cloudbuild/prombench.yaml

```diff
@@ -0,0 +1,335 @@
+# https://cloud.google.com/build/docs/build-config-file-schema#options
+options:
+  env:
+  - 'CANCEL_BENCHMARK=false'
+  - 'CLUSTER_NAME=prombench'
+  - 'CREATE_CLUSTER=true'
+  - 'DASHBOARD_ID=$_DASHBOARD_ID'
+  - 'DOMAIN_NAME='
+  - 'GITHUB_STATUS_TARGET_URL='
+  - 'GKE_PROJECT_ID=$_GKE_PROJECT_ID'
+  - 'GRAFANA_ADMIN_PASSWORD=admin'
+  - 'LAST_COMMIT_SHA='
+  - 'OAUTH_TOKEN='
+  - 'PROMBENCH_IMAGE=$_PROMBENCH_IMAGE'
+  - 'PROMETHEUS_IMAGE_REPOSITORY=gke.gcr.io/prometheus-engine/prometheus'
+  - 'PROVIDER=gke'
+  - 'SERVICEACCOUNT_CLIENT_EMAIL=$_SERVICEACCOUNT_CLIENT_EMAIL'
+  - 'WH_SECRET='
+  - 'ZONE=us-east4-a'
+  substitutionOption: ALLOW_LOOSE
+  dynamicSubstitutions: true
+
+# https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values
+substitutions:
+  _COMMENTER: $(body.comment.user.login)
+  _COMMENTER_ROLE: $(body.comment.author_association) # OWNER / MEMBER / etc. (https://docs.github.com/en/graphql/reference/enums#commentauthorassociation)
+  _GIT_REPO: $(body.repository.full_name) # GoogleCloudPlatform/prometheus
+  _PR_COMMENT_ACTION: $(body.action)
+  _PR_COMMENT_TEXT: $(body.comment.body)
+  _PR_NUMBER: $(body.issue.number)
+
+# https://cloud.google.com/build/docs/build-config-file-schema#availablesecrets
+availableSecrets:
+  secretManager:
+  - versionName: projects/${PROJECT_NUMBER}/secrets/prombench_service_account_adc_json/versions/1
```

Comment:
Currently, the secrets in the secrets manager are versioned. Maybe we can use [version aliases](https://cloud.google.com/secret-manager/docs/assign-alias-to-secret-version) so that even when secrets are rotated, the alias can point to the newer version and the "latest" version need not be updated in code.

Reply:
OK, makes sense. Also names are generic enough.